tests/test_multi_file_viewer.ts

```diff
@@ -0,0 +1,415 @@
+import MultiFileViewer from '@/components/multi_file_viewer.vue';
+import { config, mount } from '@vue/test-utils';
+import Vue from 'vue';
+import Component from 'vue-class-component';
+
+beforeAll(() => {
+    config.logModifiedComponents = false;
+});
+
+@Component({
+    template:  `<div>
+                  <multi_file_viewer
+                    ref="multi_file">
+                  </multi_file_viewer>
+                </div>`,
+    components: {
+        'multi_file_viewer': MultiFileViewer
+    }
+})
+class WrapperComponent extends Vue { }
+
+describe('MultiFileViewer.vue', () => {
+
+    test('MultiFileViewer data set to values passed in by parent', () => {
+        @Component({
+            template:  `<multi_file_viewer ref="multi_file"
+                                           :height_of_view_file="height_of_view_file_in">
+                        </multi_file_viewer>`,
+            components: {
+                'multi_file_viewer': MultiFileViewer
+            }
+        })
+
+        class WrapperComponent2 extends Vue {
+            height_of_view_file_in = { "height": "540px" };
+        }
+
+        const wrapper = mount(WrapperComponent2);
+        const multi_file_viewer = wrapper.find({ref: 'multi_file'});
+        expect(multi_file_viewer.vm.$data.files_currently_viewing.length).toEqual(0);
+        expect(multi_file_viewer.vm.$data.active_tab_index).toEqual(0);
+        expect(multi_file_viewer.vm.$data.scrollable_height).toEqual(
+            wrapper.vm.$data.height_of_view_file_in
+        );
+    });
+
+    test('When a file is added it becomes the active file/tab ', () => {
+
+        const wrapper = mount(WrapperComponent);
+        let multi_file_viewer = <MultiFileViewer> wrapper.find({ref: 'multi_file'}).vm;
```

Comment:
I'm not crazy about having two variables with more or less the same name that are two different things. Maybe try something like this instead:
```
const wrapper = mount(WrapperComponent).find({ref: 'multi_file'}); // Replaces mfv
let multi_file_viewer = <MultiFileViewer> wrapper.vm;
```